Integrations/integration-SplunkPy.yml

```diff
@@ -238,6 +256,25 @@ script:
     - contextPath: Splunk.Result
       description: The splunk search result, json array where each item is a splunk event
     description: Search Splunk for events
+  - name: splunk-submit-event
+    arguments:
+    - name: index
+      required: true
+      description: Splunk index to push data to
+    - name: data
+      required: true
+      default: true
+      description: The new event data to push
+    - name: sourcetype
+      required: true
+      description: Event sourcetype
+    - name: host
+      required: true
+      description: Event host
+    description: Create new event in splunk
+  - name: splunk-print-indexes
```

Comment:
plural form of index is indices

also maybe instead of `print` we should do `get` to be consistent with other commands?

Reply:
Not according to splunk: http://dev.splunk.com/view/python-sdk/SP-CAAAEE6#listinputs